Bottom: 50abecfa4b84607fd0f60ee9711a072c74f96fac
Top:    388ac7e6d865ab75669b0ceaa685d6219de4c5a0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 17:17:39 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
